NitroxLauncher/ServerPage.xaml

```diff
@@ -1,338 +1,58 @@
-﻿<Page
-      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+﻿<Page xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
       xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
       xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
       xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
       xmlns:local="clr-namespace:NitroxLauncher"
-      xmlns:Themes="clr-namespace:Microsoft.Windows.Themes;assembly=PresentationFramework.Aero2" x:Class="NitroxLauncher.ServerPage"
-      mc:Ignorable="d" 
-      d:DesignHeight="470" d:DesignWidth="719"
+      xmlns:Themes="clr-namespace:Microsoft.Windows.Themes;assembly=PresentationFramework.Aero2"
+      x:Class="NitroxLauncher.ServerPage"
+      mc:Ignorable="d" d:DesignHeight="470" d:DesignWidth="719"
       DataContext="{Binding RelativeSource={RelativeSource Self}}"
-      Title="Server Page" Margin="0,0,0,0">
-    <Page.Resources>
+      Title="{DynamicResource ResourceKey=NitroxServerTitle}"
+      Margin="0">
 
-        <Style x:Key="FocusVisual">
```

Comment:
Why is this considered "dead code"? Doesn't it apply any styling (or just the default)? Even if close to the default, I would assume there's purpose behind including it here.

Reply:
None of the style define in this `<Page.Resources>` is used, the combobox were replaced by radio buttons few PR ago. In my opinion keeping 200 hundreds line of codes just for a simple combobox was a bit too much, wasn't it ?

Reply:
Thanks for clarifying.